config/xml_schemas/config_machines.xsd

```diff
@@ -159,6 +160,8 @@
         <!-- MAX_TASKS_PER_NODE: maximum number of threads*tasks per
              shared memory node on this machine-->
         <xs:element ref="MAX_TASKS_PER_NODE" minOccurs="1" maxOccurs="4"/>
+        <!-- MAX_GPUS_PER_NODE: maximum number of GPUs per node on this machine-->
+        <xs:element ref="MAX_GPUS_PER_NODE" minOccurs="0" maxOccurs="4"/>
```

Comment:
Should maxOccurs here really be 1?  I see that MAX_TASKS_PER_NODE also has 4 but I think that this must be a mistake.

Reply:
Thanks @jedwards4b . I think it makes more sense to set `maxOccurs` to 1 but I just followed the setting for `MAX_TASKS_PER_NODE`. I could update it to 1 if it looks fine to you.

Reply:
Yes, please change to 1.